File: JS Exams/01.  Basics Exam Collection/Basics Online Retake Exam - 2 and 3 May 2019/01. Food Delivery/01. Food Delivery.js
function foodDelivery(chicken_numbers, fish_numbers, vegetable_numbers) {
    chicken_numbers *= 10.35
    fish_numbers *= 12.40
    vegetable_numbers *= 8.15
    let dessert = 1.20
    let delivery = 2.50

    let subtotal = chicken_numbers + fish_numbers + vegetable_numbers
    let total = subtotal * dessert + delivery

    return `Total: ${total.toFixed(2)}`
}

//console.log(foodDelivery(2, 4, 3))

File: Python Exams/01. Basic Exam Collection/Basics Online Retake Exam - 2 and 3 May 2019/01. Food Delivery/01. Food Delivery.py
chicken_numbers, fish_numbers, vegetable_numbers = [int(input()) for x in range(3)]
chicken_numbers *= 10.35
fish_numbers *= 12.40
vegetable_numbers *= 8.15
dessert = 1.20
delivery = 2.50

subtotal = chicken_numbers + fish_numbers + vegetable_numbers
total = subtotal * dessert + delivery

print(f'Total: {total:.2f}')


'''
2
4
3
'''
